src/template-cache.js

```diff
@@ -77,7 +78,7 @@ _.extend(TemplateCache.prototype, DomMixin, {
   // using a template-loader plugin as described here:
   // https://github.com/marionettejs/backbone.marionette/wiki/Using-marionette-with-requirejs
   loadTemplate(templateId, options) {
-    const $template = this.findEls(templateId);
+    const $template = Backbone.$(templateId);
```

Comment:
If `TemplateCache` solutions were popular they should have their own plugin that isn't tied to how the views use the DOM.  Like you should be able to change your `TemplateCache` solution and your `DomApi` solution independently of each other.

FWIW `TemplateCache` already has a method for determining how to load templates that is overridable and there's really only that single use, if it is even in use.  While the default implementation does use the DOM I don't think we'd suggest that it is the only place to store templates.  It's really doing a different job than views and regions.  The only thing currently using `getEl` is `Region` and I think a better implementation of Region would avoid Regions doing their own look-ups.  So I would highly bet that we'd have DomApi that would be specifically for `TemplateCache` and to me it is already redundant.